builtin/settingtypes.txt

```diff
@@ -1843,6 +1843,9 @@ chatcommand_msg_time_threshold (Chat command time message threshold) float 0.1 0
 #    A message to be displayed to all clients when the server shuts down.
 kick_msg_shutdown (Shutdown message) string Server shutting down.
 
+#    Whether to ask clients to reconnect after a shutdown request.
```

Comment:
This isn't entirely accurate, a shutdown request from Lua can still choose to use a different value.